db-engine/src/main/java/com/splicemachine/db/impl/sql/compile/GroupByNode.java

```diff
@@ -1445,4 +1445,24 @@ public String printExplainInformation(String attrDelim) throws StandardException
         return sb.toString();
     }
 
+    public HashSet<String> getNoStatsColumns() throws StandardException {
+        if (costEstimate == null)
+            throw new RuntimeException("Should not be null");
+
+        HashSet<String> noStatsColumns = new HashSet<>();
+        CollectNodesVisitor cnv = new CollectNodesVisitor(ColumnReference.class);
+        for (OrderedColumn oc : groupingList) {
+            oc.accept(cnv);
+        }
+        for (AggregateNode an : aggregateVector) {
```

Comment:
We do not need the stats on columns inside the aggregate function. So there is no need to visit the aggregate function nodes.

Reply:
Got it, removed in new commit.